### PR TITLE
Skip any Python3 build now that Python 3.4 is no more supported

### PR DESCRIPTION
**Pull request by nsoranzo**
MNT: Re-rendered with conda-smithy 2.1.1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py35]
+    skip: True  # [py3k]
```
Comment (jakirkham) [inline recipe/meta.yaml]: No need to change the `skip` here. It would be dropped anyways as we dropped Python 3.4 in `conda-smithy` 2.0.0.

Comment (nsoranzo) [inline recipe/meta.yaml]: I have changed this to skip also py36 and future Python versions, this package is needed only for Python 2, 3.3 and 3.4.

Comment (jakirkham) [inline recipe/meta.yaml]: Right. So my recommendation would be to change this to `py>34` instead. This has the same effect, but it is clearer in its meaning. Plus, if people want a working Python 3.4 environment, they can still build one locally.

Comment (nsoranzo) [inline recipe/meta.yaml]: Done, thanks @jakirkham , I was not aware of this syntax, seems to be undocumented on https://conda.io/docs/building/meta-yaml.html#preprocessing-selectors .

Comment (jakirkham) [inline recipe/meta.yaml]: Awesome, thanks.

Good point. If you have time, feel free to add a PR against the [doc repo]( https://github.com/conda/conda-docs ).

**Pull request merged**